src/ontology/cl-edit.owl

```diff
@@ -22603,6 +22602,16 @@ SubClassOf(obo:CL_0008039 obo:CL_0000100)
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_0008002))
 SubClassOf(obo:CL_0008039 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_7742))
 
+# Class: obo:CL_0008040 (squamous endothelial cell of venule)
+
+AnnotationAssertion(Annotation(rdfs:comment "GOC:tfm") obo:IAO_0000115 obo:CL_0008040 "A endothelial cell of the venule that is squamous shaped. This is in contrast to the cubodial shape of high endothelial venule cells.")
+AnnotationAssertion(dc:contributor obo:CL_0008040 "https://orcid.org/0000-0002-7073-9172")
```

Comment:
The ORCID is a literal. Please, add in the IRI editor.

Reply:
That's why the QC is failing.

Reply:
Can we remove annotation properties that we don't use anymore? It creates confusion.

Reply:
I'm not sure, we're importing OMO, which is where it comes from. It's important to make sure you use the correct annotation, and the QC is there to help you.